README.md

```diff
@@ -1,12 +1,12 @@
-[![Neat](http://images.thoughtbot.com/bourbon/neat-logo.svg)](http://neat.bourbon.io)
+![(just) Neat](http://meows.us/just-neat-logo.svg)
```

Comment:
Obviously temporary. I was playing with the idea of "just-neat" becoming the name of the package.